pallets/schemas/README.md

```diff
@@ -0,0 +1,79 @@
+
+Schemas provide for both consistent data structures, but also human intent of the message.
+Some schemas may be structurally the same, but have a different interpretation of the contents.
+For example, two schemas might both only have a hash for contents, but one is a recording of the hash for time validation purposes, while the other is to mark an off-chain vote.
+
+### Schema Parameters
+
+- Model: The actual JSON representing the data structure.
+- Model Type: The type of serialization used. (Parquet, Avro, etc...)
+- Settings: Various options for the Schema like signature requirements.
+- Payload Location: The location the data for this Schema is stored.
+
+### Mainnet vs Testnet Schema Creation
+
+Mainnet schemas must be approved by the Frequency Council.
+This is to prevent malicious schemas and increase the documentation around the schemas available.
+
+On Testnets, schemas can be created by anyone, so there is _no_ guarantees around schema correctness or quality.
+If you want to use a particular schema on a testnet, it is suggested that you use specific Schema Ids or publish the needed schema yourself.
+
+### Actions
+
+The Schemas pallet provides for:
+
+- Registering or proposing new Schemas.
+- Retrieving schemas by their Id or name.
+- Validating a Schema model.
+- Retrieving last registered Schema Id.
+
+## Interactions
+
+### Extrinsics
+
+| Name/Description                                                                                      | Caller                                          | Payment | Key Events                                                                                                                               | Runtime Added |
+| ----------------------------------------------------------------------------------------------------- | ----------------------------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `set_max_schema_model_bytes`<br />Governance action to alter the maximum byte length of Schema models | Governance                                      | Tokens  | [`SchemaMaxSizeChanged`](https://frequency-chain.github.io/frequency/pallet_schemas/pallet/enum.Event.html#variant.SchemaMaxSizeChanged) | 1             |
+| `propose_to_create_schema_v2`<br />Creates a Council proposal a new schema to the Frequency Council   | Token Account                                   | Tokens  | [`Proposed`](https://paritytech.github.io/polkadot-sdk/master/pallet_collective/pallet/enum.Event.html#variant.Proposed)                 | 66            |
+| `create_schema_via_governance_v2`<br />Governance called version of `create_schema_v3`                | Frequency Council                               | Tokens  | [`SchemaCreated`](https://frequency-chain.github.io/frequency/pallet_schemas/pallet/enum.Event.html#variant.SchemaCreated)               | 66            |
```

Comment:
I'm not sure how else to word this but it's awkward and unclear.  At any rate if you left it, it should be "Governance-called"

Reply:
I had trouble with it as well. Swapped to `Governance action` like the `set_max_schema_model_bytes` has. How does that work?